integration-tests/oidc-wiremock/src/main/resources/application.properties

```diff
@@ -170,7 +167,6 @@ quarkus.oidc.bearer-required-algorithm.credentials.secret=secret
 quarkus.oidc.bearer-required-algorithm.token.signature-algorithm=PS256
 
 quarkus.oidc.bearer-azure.provider=microsoft
-quarkus.oidc.bearer-azure.authentication.user-info-required=false
```

Comment:
This is actually what should block it even if the injection point detected. The Java test below should also stay intact. Have a look in `integration-tests/oidc-code-flow` or `integration-tests/oidc-tenancy`, one of the static or dynamic tenants enables `user-info-required=true`, so removing it is what would test this fix. (the original test will likely need to be removed)

Reply:
@michalvavrik I'm not sure what you mean with some of those comments, but anyway, overriding an explicitly configured `user-info-required=false` has never been in scope. All what we agreed to in the Discussion, was that `user-info-required` should only be set to `true` if 1) the injection point exists 2) provider supports userinfo - and `metadata.getUserInfoUri != null` is what can confirm it.
If the given tenant has `user-info-required=false` then it can mean that on the path secured by a given tenant the user info acquisition is not necessary even if a given provider supports UserInfo - you change it to required. In general we don't override internally the properties set by the user directly